Makefile

```diff
@@ -2,19 +2,33 @@ SHELL := /usr/bin/env bash
 
 TEMPLATEFILES := $(shell find ./templates -type f -name "*.yaml.tmpl")
 
+BASE_IMG="managed-cluster-validating-webhooks"
 NAMESPACE ?= openshift-validation-webhook
 SVCNAME ?= validation-webhook
 SANAME ?= validation-webhook
-IMAGETAG ?= latest
+GIT_HASH=$(shell git rev-parse --short=7 HEAD)
+IMAGETAG="${GIT_HASH}"
 CABUNDLECONFIGMAP ?= webhook-cert
 VWC_ANNOTATION ?= managed.openshift.io/inject-cabundle-from
+QUAY_USER=app-sre
 
-IMG ?= quay.io/cshereme/python3-webhookbase
+IMG ?= quay.io/${QUAY_USER}/${BASE_IMG}
+
+
+SELECTOR_SYNC_SET_TEMPLATE_DIR=deploy
+BUILD_DIRECTORY=build
+SELECTOR_SYNC_SET_DESTINATION=build/selectorsyncset.yaml
+REPO_NAME=managed-cluster-validating-webhooks
```

Comment:
```suggestion
REPO_NAME ?= managed-cluster-validating-webhooks
```

To allow overriding

Reply:
Why would we allow this to be overriden?

Reply:
I see it related to the previous suggestion: if I'm doing development locally and for some reason don't have the image in a registry named `managed-cluster-validating-webhooks`, it could be useful to change the name.

Reply:
Changing the registry  I can see.  Changing the image doesn't make sense.  Testing locally or with my own user I don't need to changing the image name.  I'd just need to push to `nmalik` in quay.io vs pushing to `app-sre` which I don't have permission to.  But I see we permit this in other places so I'll accept it with the expectation we'll *eventually* have centralized standardization for these things.